src/parameter_declarations.ts

```diff
@@ -482,6 +490,24 @@ Parameter.declare({
 		'(OPTIONAL) Provide a custom content type to all files within the upload to be used by the gateway to display the content'
 });
 
+Parameter.declare({
+	name: RemotePathParameter,
+	aliases: ['--remote-path'],
+	description: `the remote path for the file that will be uploaded
+\t\t\t\t\t\t\t• MUST be used in conjunction with --dest-file-name
+\t\t\t\t\t\t\t• Can NOT be used in conjunction with --local-file-path
+\t\t\t\t\t\t\t• Can NOT be used in conjunction with --local-files
+\t\t\t\t\t\t\t• Can NOT be used in conjunction with --local-paths
+\t\t\t\t\t\t\t• Can NOT be used in conjunction with --local-path
+\t\t\t\t\t\t\t• Can NOT be used in conjunction with --local-csv`,
+	forbiddenConjunctionParameters: [
```

Comment:
The Commander layer would handle the required parameters by itself
```suggestion
	requiredConjunctionParameters: [
		DestinationFileNameParameter
	],
	forbiddenConjunctionParameters: [
```